qdrant-landing/content/documentation/guides/installation.md

```diff
@@ -6,21 +6,74 @@ aliases:
   - ../installation
 ---
 
-# Installation options
+## Installation requirements
 
-## Docker
+### CPU and memory
 
-The easiest way to start using Qdrant is to run it from a ready-made Docker image.
-The latest versions are always available on [DockerHub](https://hub.docker.com/r/qdrant/qdrant/tags?page=1&ordering=last_updated).
+The amount of CPU and memory that Qdrant needs depends on the amount of vectors, their dimensions, the additional payload data that you store, and their indexes, as well as storage, replication and quantization configuration.
+
+Our [Cloud Pricing Calculator](https://cloud.qdrant.io/calculator) can give you a very rough approximation of the necessary resources to store the vectors without any payload or index data.
+
+### Storage
+
+For persistent storage, Qdrant requires a block storage device with a proper file system such es ext4. This is especially important, if you mount an external volume inside the Qdrant container or if you use a Kubernetes Persistent Volume.
+Qdrant won't work on a file storage device, such as NFS, or an object storage such as S3.
```

Comment:
I'll see, if we can compile a list of tested file systems.

Distributed network file systems like NFS or object storage like S3 won't work with Qdrant. Same as most other databases.

I think making the distinction between "disk-based filesystem" and "network-based filesystem" is misleading. It's perfectly fine to use network attached block storage. For example an ext4 volume mounted through iSCSI.

Reply:
It might be best to wait until you can compile a list. But if that takes time, we can "wave our hands" a bit by:

Linking to some definitions, such as:

- https://en.wikipedia.org/wiki/File_system#Disk_file_systems
- https://en.wikipedia.org/wiki/File_system#Network_file_systems 
- https://en.wikipedia.org/wiki/ISCSI

So alternatively:

```suggestion
For persistent storage, Qdrant requires block-level access to storage devices with a [Disk file system](https://en.wikipedia.org/wiki/File_system#Disk_file_systems) format. Network systems such as [iSCSI](https://en.wikipedia.org/wiki/ISCSI) that provide block-level access are also acceptable.
Qdrant won't work with [Network file systems](https://en.wikipedia.org/wiki/File_system#Network_file_systems) such such as NFS, or [Object storage](https://en.wikipedia.org/wiki/Object_storage) such as S3.
```

Reply:
Proper means POSIX in this ocntext

Reply:
Thanks @generall! So maybe:

```suggestion
For persistent storage, Qdrant requires block-level access to storage devices with a [POSIX-compatible file system](https://www.quobyte.com/storage-explained/posix-filesystem/). Network systems such as [iSCSI](https://en.wikipedia.org/wiki/ISCSI) that provide block-level access are also acceptable.
Qdrant won't work with [Network file systems](https://en.wikipedia.org/wiki/File_system#Network_file_systems) such such as NFS, or [Object storage](https://en.wikipedia.org/wiki/Object_storage) systems such as S3.
```